tests/app/validation/test_sum_check_validator.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+from unittest.mock import Mock
+
+from wtforms.validators import ValidationError
+
+from app.jinja_filters import format_number
+from app.validation.error_messages import error_messages
+from app.validation.validators import SumCheck
+
+
+class TestSumCheckValidator(unittest.TestCase):
+    """
+    Sum check validator evaluates a calculated total against a target total, given a condition
+    """
+    def test_equal_condition_check(self):
+        validator = SumCheck()
+
+        mock_form = Mock()
+
+        condition = 'equal-to'
+        calculation_total = 10
+        target_total = 11.5
+        label = 'Total'
+
+        with self.assertRaises(ValidationError) as ite:
+            validator(mock_form, condition, calculation_total, target_total, label)
+
+        self.assertEqual(error_messages['SUM_NOT_EQUAL_TO_TOTAL'] % {'label': label,
+                                                                     'total': self.format_min_max(target_total)},
+                         str(ite.exception))
+
+    def test_less_or_equal_condition_check(self):
+        validator = SumCheck()
+
+        mock_form = Mock()
+
+        condition = 'less-than-equal-to'
+        calculation_total = 20
+        target_total = 11.5
+        label = 'Total'
+
+        with self.assertRaises(ValidationError) as ite:
+            validator(mock_form, condition, calculation_total, target_total, label)
+
+        self.assertEqual(error_messages['SUM_NOT_EQUAL_TO_OR_LESS_THAN_TOTAL'] % {'label': label,
+                                                                                  'total': self.format_min_max(
+                                                                                      target_total)},
+                         str(ite.exception))
+
+    def test_less_than_condition_check(self):
+        validator = SumCheck()
+
+        mock_form = Mock()
+
+        condition = 'less-than'
+        calculation_total = 11.99
+        target_total = 11.5
+        label = 'Total'
+
+        with self.assertRaises(ValidationError) as ite:
+            validator(mock_form, condition, calculation_total, target_total, label)
+
+        self.assertEqual(error_messages['SUM_NOT_LESS_THAN_TOTAL'] % {'label': label,
+                                                                      'total': self.format_min_max(target_total)},
+                         str(ite.exception))
+
+    @staticmethod
```

Comment:
This method already exists in `app.validation.validators`. Duplicating it here doesn't assert anything.